src/include/daos/common.h

```diff
@@ -233,8 +233,8 @@ int daos_array_find(void *array, unsigned int len, uint64_t key,
 		    daos_sort_ops_t *ops);
 void daos_array_shuffle(void *arr, unsigned int len, daos_sort_ops_t *ops);
 
-int  daos_sgl_init(d_sg_list_t *sgl, unsigned int nr);
-void daos_sgl_fini(d_sg_list_t *sgl, bool free_iovs);
+#define daos_sgl_init d_sgl_init
+#define daos_sgl_fini d_sgl_fini
```

Comment:
This is technical debt, I would prefer to do a search-and-replace across the codebase to remove this, although I accept it would probably cause a number of conflicts.

Reply:
ok.